app/controllers/admin/api/api_docs_services_controller.rb

```diff
@@ -132,8 +133,20 @@ def api_docs_params(*extra_params)
     params.require(:api_docs_service).permit(*permit_params)
   end
 
+  def api_docs_services
+    collection = current_account.api_docs_services.accessible
+    return collection unless current_user
+    collection.permitted_for(current_user)
+  end
+
   def find_api_docs_service
-    @api_docs_service = current_account.api_docs_services.find(params[:id])
+    @api_docs_service = api_docs_services.find(params[:id])
```

Comment:
Why not change this to `params.require(:id)`

Reply:
This param is in the URL. It will always be present or the system will respond with a 404.